tests/payment/001/a/apayment.spec.js

```diff
@@ -0,0 +1,87 @@
+"use strict";
+// import { formABCD_input, formA_input, formB_input, formACD_input, formCD_input } from 'payment.library.js';
+// var myModule = require('./payment.library.js');
+
+const Nightmare = require( "nightmare" ),
+      expect = require( "chai" ).expect,
+      BASE_URL = "https://www.kidguard.com/funnel/payment/011/a?first_name=Vernon&last_name=Swanson&email=zinoto@mailinator.com&phone=+921-79-3649008&site=cellmon&log_id=12574785&password=Pa$$w0rd!&password_verify=Pa$$w0rd!&secret_question_answer=Reprehenderit%20sunt%20voluptatibus%20non%20repudiandae%20q&secret_question=What%20was%20your%20first%20grade%20teacher%27s%20name%3F"
```

Comment:
you can try to use `querystring` lib to simplify the generation of the url. please refer to https://nodejs.org/api/querystring.html